here-naksha-lib-core/src/main/java/com/here/naksha/lib/core/models/geojson/implementation/XyzProperties.java

```diff
@@ -54,7 +54,7 @@ public class XyzProperties extends JsonObject {
    * Properties used for the base layer (used in consistent store).
    */
   @AvailableSince(NakshaVersion.v2_0_7)
-  public static final String HERE_META_NS = "@ns:com:here:meta";
+  public static final String HERE_META_NS = "@ns:com:here:mom:meta";
```

Comment:
Was this change discussed? This change might have some consequences in other parts

Reply:
yes, this is agreed change as per MOM. What was set earlier was incorrect.